SpecFlowProject10/Hooks/Hooks1.cs

```diff
@@ -0,0 +1,33 @@
+﻿using TechTalk.SpecFlow;
+using Test.Form;
+using System.Drawing.Text;
+using Test.Hooks;
+namespace UnitTestProject1.Hooks
+{
+    [Binding]
+    public sealed class Hooks1
```

Comment:
Why Hooks1?

Reply:
do I need to change the class/file name?